locale/pl/LC_MESSAGES/javascript.po

```diff
@@ -32,7 +33,8 @@ msgstr "Wybierz plik…"
 
 #: static/js/common/upload-addon.js:60
 msgid "Your add-on should end with .xpi, .jar or .xml"
-msgstr "Nazwa pliku dodatku powinna kończyć się rozszerzeniem .xpi, .jar lub .xml"
+msgstr ""
+"Nazwa pliku dodatku powinna kończyć się rozszerzeniem .xpi, .jar lub .xml"
```

Comment:
Should we merge then? I know you were usually doing it in the past, should I now do it? Or @andymckay ?
Are there things to look for, double check, be aware of, be cautious of?